packages/mask/src/plugins/NFTCard/SNSAdaptor/NFTCardDialog.tsx

```diff
@@ -0,0 +1,71 @@
+import { useChainIdValid } from '@masknet/plugin-infra/web3'
+import { InjectedDialog } from '@masknet/shared'
+import { MaskTabList, useTabs } from '@masknet/theme'
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { CrossIsolationMessages } from '@masknet/shared-base'
+import { TabContext } from '@mui/lab'
+import { DialogContent, Tab } from '@mui/material'
+import { useEffect, useState } from 'react'
+import { NFTCardDialogUI } from './NFTCardDialogUI'
+import { useStyles } from '../useStyles'
+import { useNFTCardInfo } from './hooks/useNFTCardInfo'
+import { useI18N } from '../../../utils'
+
+export enum NFTCardDialogTabs {
+    About = 'About',
+    Offers = 'Offers',
+    Activity = 'Activity',
+}
+
+export function NFTCardDialog() {
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    const [tokenId, setTokenId] = useState('')
+    const [tokenAddress, setTokenAddress] = useState('')
+    const chainIdValid = useChainIdValid(NetworkPluginID.PLUGIN_EVM)
+    const [open, setOpen] = useState(false)
+    const { asset, orders, events, rarity, provider, setProvider } = useNFTCardInfo(tokenAddress, tokenId)
+
+    const [currentTab, onChange] = useTabs<NFTCardDialogTabs>(
+        NFTCardDialogTabs.About,
+        NFTCardDialogTabs.Offers,
+        NFTCardDialogTabs.Activity,
+    )
+
+    useEffect(() => {
+        if (!chainIdValid) setOpen(false)
+        return CrossIsolationMessages.events.requestNFTCardDialog.on(({ open, address, tokenId }) => {
+            setOpen(open)
+            setTokenAddress(address)
+            setTokenId(tokenId)
+        })
+    }, [chainIdValid])
+
+    return (
+        <TabContext value={currentTab}>
+            <InjectedDialog
+                open={open}
+                title={t('plugin_collectible_nft_detail')}
```

Comment:
This is a new plugin, use plugin scope i18n. 
You can follow this guide https://github.com/DimensionDev/Maskbook/wiki/Draft:-Plugin-scope-i18n-guide

BTW, `useI18n` can coexist with `useBaseI18n`